inc/settings.php

```diff
@@ -60,6 +68,75 @@ function update_privacy_policy_page() {
 	update_option( $privacy_option, $updated_id );
 }
 
+/**
+ * Auto-create the policy page based on the button that was clicked.
+ *
+ * @return void
+ */
+function create_policy_page() {
+	if (
```

Comment:
Think this also needs a `check_admin_referer()` check

Reply:
`check_admin_referer` breaks the admin here (maybe because of when the action is being fired), so adding a nonce check instead.